[IMP] Adicionados campos de endereço da localização na empresa.

cmsalmeida opened the pull request:
Modificados:
  1- l10n_br_base
  2- l10n_br_zip
  No l10n_br_base: Adicionei os campos de endereço da localização Brasil l10n_br_base à empresa. Mantém a sincronização com o res_partner respetivo.
  No l10n_br_zip: Adicionei o botão de pesquisa na empresa.

Commit:
[IMP] Adicionados campos de endereço da localização na empresa.
  Modificados:
  1- l10n_br_base
  2- l10n_br_zip
  No l10n_br_base: Adicionei os campos de endereço da localização Brasil l10n_br_base à empresa. Mantém a sincronização com o res_partner respetivo.
  No l10n_br_zip: Adicionei o botão de pesquisa na empresa.

mileo commented:
Boa! Amanhã vou testar e posto aqui de novo!

cmsalmeida commented:
Valeu ;)

renatonlima commented:
Olá Carlos,

Hoje mais tarde vou testar os pull resquests aberto e integrados!

Valeu pela contribuição! :D

mileo commented:
Tava preparando uma instância ontem e vc já tinha até feito o commit,

root@xxxxxx:/opt/openerp/l10n_br_core# git pull
Already up-to-date.

=D

Vou popular os dados dessa empresa daqui a pouco... Se for o caso já coloco
em produção.

[]s

2013/10/24 cmsalmeida notifications@github.com

> Valeu ;)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/openerpbrasil/l10n_br_core/pull/19#issuecomment-27007596
> .

## 

Luis Felipe Miléo
skype: luisfelipemileo
http://mileo.co http://blog.mileo.co
https://launchpad.net/~mileo
https://github.com/mileo

Commit:
correções indicadas pelo Renato Lima.
adicionei os campos legal_name, cnpj_cpf, incrição estadua, municipal e suframa

Commit:
adicionei o partner_fiscal_type_id no l10n_br_account

Commit:
corrigi o nome do field partner_fiscal_type_id no l10n_br_account

cmsalmeida commented:
Fiz as alterações propostas nos comentários do Renato!

renatonlima commented:
Carlos,

Só tem um detalhe não sei se você notou mas no seu commit tem uns arquivos com a extensão .moved no módulo l10n_br_zip, você pode remove-los? em seguida faço o commit porque o restante ficou bom! :)

Commit:
removed .moved files

mileo commented:
Demorei mas testei ficou otimo, a validação dos campos esta certinha.

Se os dados são alterados no parceiro ou na empresa sempre ficam em conformidade.

Vlw

Abraços
